File: lab/c/tecnicas/valor_absoluto.c
#include <stdio.h>
#include <stdlib.h>


int main(){
    int numero;

    scanf("%d", &numero);
    printf("%d\n", abs(numero));

    scanf("%d", &numero);
    if (numero >= 0){
        printf("%d\n", numero);
    } else {
        printf("%d\n", -numero);
    }

    return 0;
}

File: lab/javascript/tecnicas/valor_absoluto.js
const prompt = require('prompt-sync')();

// Versão 1
let numero = parseInt(prompt());
console.log(Math.abs(numero));

// Versão 2
numero = parseInt(prompt());
if (numero >= 0){
    console.log(numero);
} else {
    console.log(-numero);
}
